src/lib/touchFriendlyUserbox.js

```diff
@@ -0,0 +1,41 @@
+//  Tweak touch behavior for userbox
```

Comment:
This file may live within `client/js/lib/core/`.

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/71/commits/d90277fdb90ee4349783e5c1e51137121a6e26c6
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/666